### ISSUE TITLE
Vocup shouldn't accept an empty entry

### ISSUE THREAD

**Issue by jdelange22** [daniel-lerch/vocup#64]
Like the title says. I noticed that pressing just enter or clicking 'continue' treats the input as wrongly practiced. I believe it should ignore the empty input and not continue to the next word. Perhaps a warning message to be shown like "input required".

**Comment by daniel-lerch** [comment]
I believe this is intentional. Just like in a vocabulary test at school, giving no answer is equal to a completely wrong answer. We could give user an option here to configure whether no input should be `PracticeResult.WronglyPracticed` (default) or `PracticeResult.NotPracticed` (skipped).

**Comment by jdelange22** [comment]
True, but on a computer it is likely to make a mistake, pressing enter accidentally instead of shift for example. I like the idea of adding an option. We also have another issue that involves expanding the options (#60).
